lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1457,11 +1459,12 @@ mkApiTransactionFromInfo
     -> TransactionInfo
     -> m (ApiTransaction n)
 mkApiTransactionFromInfo ti (TransactionInfo txid ins outs ws meta depth txtime txmeta) = do
-    apiTx <- mkApiTransaction ti txid (drop2nd <$> ins) outs ws (meta, txtime) txmeta $
-        case meta ^. #status of
+    apiTx <- mkApiTransaction ti txid (drop2nd <$> ins) outs ws (meta, txtime) txmeta
+        (case meta ^. #status of
```

Comment:
Funny thing... I couldn't move this stuff into a let or where binding without getting compile errors